wrapper/wrapper.go

```diff
@@ -10,15 +10,26 @@ import (
 	"os/signal"
 	"syscall"
 	"time"
+	"encoding/json"
 
 	"github.com/domsolutions/gopayloader/config"
 	"github.com/domsolutions/gopayloader/pkgs/payloader"
 )
 
-func RunGoPayLoader(reqURI, mTLScert, mTLSKey string, disableKeepAlive bool, reqs int64, conns uint, totalTime time.Duration, skipVerify bool, readTimeout, writeTimeout time.Duration, method string, verbose bool, ticker time.Duration, jwtKID, jwtKey, jwtSub, jwtIss, jwtAud, jwtHeader string, headers []string, body, bodyFile string, client string) error {
+// Converts jwtCustomClaimsJSON from string to map[string]string
+func JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON string) map[string]string {
+	jwtCustomClaimsMap := map[string]string{}
+
+  json.Unmarshal([]byte(jwtCustomClaimsJSON), &jwtCustomClaimsMap)
+  return jwtCustomClaimsMap
+}
+
+func RunGoPayLoader(reqURI, mTLScert, mTLSKey string, disableKeepAlive bool, reqs int64, conns uint, totalTime time.Duration, skipVerify bool, readTimeout, writeTimeout time.Duration, method string, verbose bool, ticker time.Duration, jwtKID, jwtKey, jwtSub, jwtCustomClaimsJSON, jwtIss, jwtAud, jwtHeader string, headers []string, body, bodyFile string, client string) error {
 	ctx, cancel := context.WithCancel(context.Background())
 	defer cancel()
 
+	jwtCustomClaimsMap := JwtCustomClaimsJSONStringToMap(jwtCustomClaimsJSON)
```

Comment:
Can you check if `jwtCustomClaimsJSON` is empty and ignore as should be an optional field

Reply:
I added the check in the `JwtCustomClaimsJSONStringToMap()` function since the `jwtCustomClaimsMap` variable is required in the config. `JwtCustomClaimsJSONStringToMap()` now just returns nil if `jwtCustomClaimsJSON` is empty